readme.md

```diff
@@ -25,7 +25,7 @@ Amas is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym).
 - [Wenjie Fan](https://github.com/DIYgod/ama) - Front-end Engineer for Linux Deepin, blogger of anotherhome.net.
 - [Maxime Euzière](https://github.com/xem/ama) - Front-end hacker and JS code-golfer.
 - [Robert Haritonov](https://github.com/operatino/ama) - Front-end Team Lead, [SourceJS](http://sourcejs.com) maintainer and Living Style Guide evangelist.
-
```

Comment:
Weird. I think GitHub's diff is not doing a job here :confused: This line wasn't actually removed at all if you compare the raw markdown files.